libs/features/mas/src/hydrate.js

```diff
@@ -123,10 +123,25 @@ export function processPrices(fields, merchCard, mapping) {
   appendSlot('prices', fields, merchCard, mapping); 
 }
 
+function parseHtml(html) {
+  const parser = new DOMParser();
+  return parser.parseFromString(html, 'text/html');
+}
```

Comment:
I dont see this getting reused, and only called in `isHtmlEmpty`. Is there a reason not to instatiate the parser and parse within `isHtmlEmpty` itself, it will technical save 3 lines of code.